src/ssl.c

```diff
@@ -1205,6 +1205,27 @@ int wolfSSL_set_ConnectFilter(
 #endif /* WOLFSSL_WOLFSENTRY_HOOKS */
 
 #ifndef WOLFSSL_LEANPSK
+#ifdef WOLFSSL_WRAPPER_ALLOCATORS
+void* wolfSSL_dtls_create_peer(int port, char* ip)
+{
+    struct sockaddr_in *addr;
+    addr = (struct sockaddr_in*)XMALLOC(sizeof(*addr), NULL, DYNAMIC_TYPE_SOCKADDR);
```

Comment:
Must check for malloc failure here... `if (addr == NULL)`.

Reply:
Done.